ios/FluentUI/HUD/HeadsUpDisplayTokenSet.swift

```diff
@@ -47,13 +36,22 @@ public class HeadsUpDisplayTokenSet: ControlTokenSet<HeadsUpDisplayTokenSet.Toke
                 }
 
             case .activityIndicatorColor:
+                let colorToken = theme.aliasTokens.colors[.foregroundLightStatic]
+                let lightColorValue = ColorValue(r: colorToken.light.r,
+                                                 g: colorToken.light.g,
+                                                 b: colorToken.light.b,
+                                                 a: 0.4)
+                let darkColorValue = ColorValue(r: colorToken.dark?.r ?? colorToken.light.r,
+                                                g: colorToken.dark?.g ?? colorToken.light.g,
+                                                b: colorToken.dark?.b ?? colorToken.light.b,
+                                                a: 0.6)
```

Comment:
Looks like the design has light and dark mode both using `FG Light Static` @ different alphas.

Reply:
That's how it's implemented as well but it's pulling its dark `ColorValue` for dark mode. 

<img width="352" alt="Screen Shot 2022-09-28 at 2 57 39 PM" src="https://user-images.githubusercontent.com/106181067/192896301-6103c562-bdc7-42fc-b98f-4e265c817da5.png">